src/app/_components/supported-chains-section.tsx

```diff
@@ -8,50 +8,56 @@ import {
   ZkSyncIcon,
   LineaIcon,
   ModeIcon,
+  BlastIcon,
 } from "@/app/_components/icons";
 import { Text } from "@/app/_components/text";
 
 const chains = [
   {
     label: "Ethereum Mainnet",
     Icon: EthereumIcon,
-    containerClassName: "col-start-1",
+    containerClassName: "sm:col-start-1",
```

Comment:
ooc - is this something we can programmatically generate?

Reply:
Not sure at this point tbh 🤔 I think we might actually need new designs at some point